openapi/schemas/processes-job-management/jobList.yaml

```diff
@@ -0,0 +1,13 @@
+type: object
+required:
+  - jobs
+  - links
+properties:
+  jobs:
+    type: array
+    items:
+      $ref: "statusInfo.yaml"
```

Comment:
statusInfo.yaml sounds a bit misleading. Shouldn't this be called jobDetails.yaml or similar?

Reply:
This uses the original OAP Core reference https://github.com/opengeospatial/ogcapi-processes/blob/master/openapi/schemas/processes-core/statusInfo.yaml
Same name is employed to make mapping more obvious.

The `GET /jobs` endpoint is allowed to return very minimalistic info (eg: only the job `id`, `type` and `status` is sufficient). A "Job Details" would make more sense for `GET /jobs/{jobId}` which _**should**_ return much more information, but is also allowed with only the same minimal set of 3 properties for backward compatibility.

Reply:
openEO requires created in addition, but otherwise it's the same (except for type for now).